### PR TITLE
Absolute imports and CSS Modules

### PR DESCRIPTION
**Pull request by WadadParker**
Have configured absolute imports and CSS Modules in this branch.

### CODE REVIEW

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styles from "src/pages/home.module.css";
+
+export const Home=()=>
+{
+    return (
+        <div>
```
Comment (HeyNitin) [inline src/pages/Home.jsx]: Remove these home.jsx and home.module.css files. It's a template, we don't want to have files which anyone would need to delete after cloning.

File: src/backend/db/users.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ import { formatDate } from "../utils/authUtils";
 export const users = [
   {
     _id: uuid(),
-    firstName: "Adarsh",
-    lastName: "Balika",
-    username: "adarshbalika",
```
Comment (HeyNitin) [inline src/backend/db/users.js]: Change these back to default, let's use generic names only

**Review by HeyNitin — CHANGES_REQUESTED** [review]
left some comments, also while it's perfectly fine to use npm if you were facing any troubles with yarn, This was set up with yarn initially so let's not pollute the template with both package managers. Try with yarn if possible.

File: package.json [hunk 3]
```diff
@@ -16,6 +16,7 @@
     "jwt-encode": "^1.0.1",
     "miragejs": "^0.1.47",
     "react": "^18.2.0",
+    "react-css-modules": "^4.7.11",
```
Comment (HeyNitin) [inline package.json:19]: Why do we need this?

Comment (WadadParker) [reply package.json:19]: When I initially did it without installing this, it gave me a lot of error, I then read that we need to install these dependencies for css modules to work in react.

**Pull request closed without merging**